tox.ini

```diff
@@ -63,10 +63,10 @@ commands =
 
 setenv =
     COVERAGE_FILE=.coverage.{envname}
-    Plone43: VERSION=4.3-latest
-    Plone50: VERSION=5.0-latest
-    Plone51: VERSION=5.1-latest
-    Plone52: VERSION=5.2-latest
+    Plone43: VERSION=4.3
+    Plone50: VERSION=5.0
+    Plone51: VERSION=5.1
+    Plone52: VERSION=5.2
```

Comment:
... not sure about these changes.

Reply:
smae here. @balavec do you have a reason to change this? If not i would keep it like it was.